### PR TITLE
[main] Bump min K8s version to 1.25

### PR DESCRIPTION
**Pull request by ReToCode**
As per title.

### CODE REVIEW

File: olm-catalog/serverless-operator/project.yaml [hunk 1]
```diff
@@ -23,13 +23,13 @@ requirements:
...
+        minVersion: 1.25.0
     golang: '1.21'
     nodejs: 16.x
     ocpVersion:
-        min: '4.11'
-        max: '4.14'
-        label: 'v4.11'
+        min: '4.12'
+        max: '4.15'
+        label: 'v4.12'
```
Comment (matzew) [inline olm-catalog/serverless-operator/project.yaml:32]: these are the only changes, rest is from the `make generated-files`, right?

(asking b/c it is just one commit)

**Review by matzew — APPROVED** [review]
/lgtm
/approve

/hold
(you can never be sure w/ CI ... :trollface: )

File: olm-catalog/serverless-operator/project.yaml (hunk 1 above)
Comment (ReToCode) [reply olm-catalog/serverless-operator/project.yaml:32]: Yepp

**Pull request merged**